### PR TITLE
Update configuration to use the new names and version 1.0.0: MAISTRA-678

### PR DESCRIPTION
**Pull request by knrc**

### CODE REVIEW

File: helm/istio/charts/istio_cni/templates/daemonset.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ spec:
     metadata:
       labels:
         heritage: {{ .Release.Service }}
-        maistra-version: 0.12.0
+        maistra-version: 1.0.0
         release: {{ .Release.Name }}
```
Comment (jwendell) [inline helm/istio/charts/istio_cni/templates/daemonset.yaml:26]: this label is added automatically by patch-charts.sh, no?

Comment (knrc) [reply helm/istio/charts/istio_cni/templates/daemonset.yaml:26]: It wasn't but should be, the script hadn't been updated to do that

File: helm/istio/charts/istio_cni/templates/daemonset.yaml [hunk 2]
```diff
@@ -9,7 +9,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     heritage: {{ .Release.Service }}
-    maistra-version: 0.12.0
```
Comment (brian-avery) [inline helm/istio/charts/istio_cni/templates/daemonset.yaml:12]: We auto-generate the version that the rest of Istio in the build process. Would it make sense to update these there as well?

**Review by brian-avery — APPROVED** [review]

File: helm/istio/charts/istio_cni/templates/daemonset.yaml (hunk 2 above)
Comment (rcernich) [reply helm/istio/charts/istio_cni/templates/daemonset.yaml:12]: Yes, I think these should be removed.  See @knrc's comment below.

Comment (brian-avery) [reply helm/istio/charts/istio_cni/templates/daemonset.yaml:12]: Ah ok. I misunderstood that one as talking about the helm labels.

Comment (knrc) [reply helm/istio/charts/istio_cni/templates/daemonset.yaml:12]: The script does not currently modify this version, I'll push an update which handles this.

File: helm/istio/charts/istio_cni/templates/daemonset.yaml (hunk 1 above)
Comment (knrc) [reply helm/istio/charts/istio_cni/templates/daemonset.yaml:26]: @jwendell you are correct the script does add in the label, the generated version contained two maistra-version labels so the daemonset originally had the 0.12.0 and 1.0.0 versions.  I'll tidy up the overlay version to fix this.

File: helm/istio/charts/istio_cni/templates/daemonset.yaml (hunk 2 above)
Comment (knrc) [reply helm/istio/charts/istio_cni/templates/daemonset.yaml:12]: I looked deeper into this and the script does handle it, the original overlay therefore resulted in two maistra-version labels being included within the daemonset

**Review by brian-avery — APPROVED** [review]

**Review by jwendell — APPROVED** [review]

**Review by rcernich — APPROVED** [review]

**Pull request merged**